index.html

```diff
@@ -22,16 +22,13 @@ <h1>Guessing Game</h1>
         <p class="message" id="number-of-guesses"></p>
         <p class="message" id="too-high">You guessed too high. Try again.</p>
         <p class="message" id="too-low">You guessed too low. Try again.</p>
-        <p class="message" id="max-guesses">
-          You reached the max number of guesses
-        </p>
-        <p class="message" id="correct">
-          Congratulations, You guessed correctly! <br />
-          Would you like to play again?
-        </p>
+        <p class="message" id="max-guesses">You reached the max number of guesses</p>
```

Comment:
When you reach the max number of guesses, you're not currently displaying this message to the user:


https://user-images.githubusercontent.com/9094098/193882063-3150046f-1958-41ca-86d7-123be7a18709.mov